docs/src/pages/lab/slider/CustomIconSlider.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Slider from '@material-ui/lab/Slider';
+import LensIcon from '@material-ui/icons/Lens';
```

Comment:
Would LensOutlined work so as so as to be distinct from the default?

Reply:
![mjlwiyfb1h](https://user-images.githubusercontent.com/16651811/45378209-46503f00-b616-11e8-90a4-7e6967bde3e8.gif)

This is how that would look like?

Reply:
I wondered about that. Not ideal... 

What about "adjust"? Otherwise "favorite" with a step slider as a sort of rating slider.

Reply:
Should it have a default background color? e.g white?

Reply:
With a default white backgroundColor it would look like this.

![dvvcfnpye3](https://user-images.githubusercontent.com/16651811/45379166-c5467700-b618-11e8-8b2a-63fc6a84498c.gif)

Reply:
That'll work (for the purpose of a demo)

Reply:
For this I have to change the `thumbIconWrapper` in my slider.js file

```
thumbIconWrapper: {
      backgroundColor: '#fff',
    },
```

Or should i override it with `classes` prop in the demo for this?

Reply:
I have added it in the demo, is that okay? 😅  @mbrookes